README.md

```diff
@@ -15,3 +15,16 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## Answer 
+
+The dominant operations of this algorithm are the outer loop and the DFS traversal. The outer loop iterates over each node and initiates a DFS for any unvisited node. This ensures that DFS is only started for nodes that have not been visited. DFS explores all the nodes connected to the `startNode` and marks them as `visited`. Since each node is visited only once, and each edge is processed only once during the DFS. The worst-case time complexity is $\Theta(V + E)$.  
+
+## Plagiarism Acknowledgement
+
+This assignment seemed pretty straight forward. The parts I had the most trouble with was implementing the algorithm using DFS and the time analysis. When implementing the algorithm I looked at two repositories to guide me through my implementation. https://github.com/COSC3020/detecting-cycles-KobeLimon21, and https://github.com/COSC3020/detecting-cycles-IshitaPatel18. These were very helpful with the analysis as well. 
```

Comment:
So what parts are from these repositories and what parts did you implement?

Reply:
I hope you are enjoying the Thanksgiving holiday. I updated my acknowledgement to include more detail about specific sections from the listed repositories I benefited from using.